src/main/kotlin/com/ampnet/mailservice/service/impl/UserMailServiceImpl.kt

```diff
@@ -68,30 +70,30 @@ class UserMailServiceImpl(
         FailedDeliveryMail(mailSender, applicationProperties, linkResolverService)
     }
 
-    override fun sendConfirmationMail(email: String, token: String, coop: String) =
-        confirmationMail.setData(token, coop).sendTo(email)
+    override fun sendConfirmationMail(request: MailConfirmationRequest) =
+        confirmationMail.setData(request.token, request.coop).sendTo(request.email, request.language)
 
-    override fun sendResetPasswordMail(email: String, token: String, coop: String) =
-        resetPasswordMail.setData(token, coop).sendTo(email)
+    override fun sendResetPasswordMail(request: ResetPasswordRequest) =
+        resetPasswordMail.setData(request.token, request.coop).sendTo(request.email, request.language)
 
     override fun sendOrganizationInvitationMail(request: OrganizationInvitationRequest) =
         invitationMail.setData(request.organization, request.coop)
-            .sendTo(request.emailsList.toList()) { failedMails ->
+            .sendTo(request.emailsList.toList(), request.language) { failedMails ->
                 val filedMailRecipients = failedMails.map { it.allRecipients.toString() }
                 failedDeliveryMail.setData(filedMailRecipients).sendTo(request.senderEmail)
             }
```

Comment:
add language to failedDeliveryMail. 
```suggestion
failedDeliveryMail.setData(filedMailRecipients).sendTo(request.senderEmail, request.language)
```